admin/css/debug-toolbar/toolbar.scss

```diff
@@ -36,6 +39,9 @@
     clear: both;
     text-align: center;
 
+    // Cursor
```

Comment:
No need for this comment, in my opinion.

Reply:
Done